controllers/user.go

```diff
@@ -38,6 +39,7 @@ func (c *UserController) actionPolicyMapping() {
 func (c *UserController) New() {
 	web.ReadFromRequest(&c.Controller)
 
+	c.Data["XSRFData"] = template.HTML(c.XSRFFormHTML())
```

Comment:
Avoid using data as it lacks clarity. How about reusing the actual naming `XSRFFormHTML`:

```suggestion
	c.Data["XSRFForm"] = template.HTML(c.XSRFFormHTML())
```